posts/2016-11-10-going-serverless-at-bandlab.md

```diff
@@ -42,34 +46,36 @@ While still at the Dutch company we managed to set up a solid framework to autom
 
 >There was only one thing we couldn’t change: **Client applications!**
 
-Every new client needed to be educated, and migrating them to AWS in general was a long process. For the most part, the refactoring of their application code was needed in order to become [12-factor-ready](https://12factor.net/).
+Every new client needed to be educated. And migrating them to AWS in general was a long process. For the most part, the refactoring of their application code was needed in order to become [12-factor-ready](https://12factor.net/).
 
 >[AWS Lambda](https://aws.amazon.com/lambda/details/) lets you run code without provisioning or managing servers. You pay only for the compute time you consume — there is no charge when your code is not running.
 
 When AWS first [introduced](https://www.youtube.com/watch?v=9eHoyUVo-yg) Lambda at re:Invent in 2014, I was a bit skeptical. Their example used S3 events. The triggering of image resizing with Lambda was awesome, but I didn’t see our clients adopting this anytime soon. 
 
-Internally we started sprinkling in custom [Lambda-Backed Custom Resources](https://aws.amazon.com/blogs/aws/aws-cloudformation-update-lambda-backed-custom-resources-more/) within CloudFormation stacks to fill-up missing gaps. This was when I started to realise how powerful Lambda actually is. We were able to automate a lot of use-cases and 3rd party applications/providers with the use of AWS Lambda.
+Internally we started sprinkling in custom [Lambda-Backed Custom Resources](https://aws.amazon.com/blogs/aws/aws-cloudformation-update-lambda-backed-custom-resources-more/) within CloudFormation stacks to fill-up missing gaps. This was when I started to realize how powerful Lambda actually is. We were able to automate a lot of use-cases and 3rd party applications/providers with the use of AWS Lambda.
+
+![pexels-photo-38125](https://cloud.githubusercontent.com/assets/20538501/20605146/70cadfb6-b22f-11e6-8252-80593cfef88d.jpeg)
```

Comment:
Please remove the image at this section, as this still refers about the previous company I worked at(has nothing to do with audio). If you would like to add an image somewhere, please use this one https://blog.bandlab.com/wp-content/uploads/2016/11/new_33b.jpg which comes directly from our BandLab blog.